src/generator/synth.ts

```diff
@@ -30,11 +33,27 @@ export interface Changelog {
   semverity: Semverity;
 }
 
-export async function synth() {
+export const gfs = {
+  rimraf: util.promisify(rimraf),
```

Comment:
`fs.rmdir` has `recursive: true` now.

Reply:
const {rmdir} = require('fs').promises;
await rmdir('blerg', {recursive: true});

Reply:
So this is completely your call, but the docs call it out as experimental / stage 1:
https://nodejs.org/api/fs.html#fs_fs_rmdir_path_options_callback

...  should we do it?

Reply:
It's been there for 6 months, I feel comfortable using it.

Reply:
If you don't mind, I'll cut a separate PR where I remove `rimraf` then.  We use it in other places in the codebase, and I'd like to track that all together.

Reply:
yeah, not a blocker.